edges/4.1/4.1.0-rc.6/4.1.0-rc.5.json

```diff
@@ -0,0 +1,7 @@
+{
+  "channels": [
```

Comment:
Why are edges associated with channels?

Reply:
> Why are edges associated with channels?

Setting up for:

```console
$ cat edges/4.1/4.1.15/4.1.14-prerelease.json 
{
  "channels": [
    "prerelease-4.1"
  ],
  "from": "4.1.14",
  "to": "4.1.15",
  "metadata": {
    "start": "2019-09-16T20:00Z",
    "duration": 360,
  }
}
$ cat edges/4.1/4.1.15/4.1.14-stable.json 
{
  "channels": [
    "stable-4.1"
  ],
  "from": "4.1.14",
  "to": "4.1.15",
  "metadata": {
    "start": "2019-09-17T20:00Z",
    "duration": 360,
  }
}
```

or similar to stage the stable edge rollout a day after the prerelease rollout (cf. https://github.com/coreos/fedora-coreos-streams/pull/10).  And sharing metadata about start dates here so we don't have to have multiple Cincinnati pods communicating with each other about when they decided to start the rollout.

Reply:
Hmm. I guess I would expect the channels to be associated with nodes instead. Can the channel be specified on both edges and nodes? What happens if they differ?

Reply:
It will be a CI error in this repo to define an edge in a channel where either node does not belong to that channel.

Reply:
I see. So this is an edge whitelist rather than our current blacklist approach.

Reply:
> So this is an edge whitelist rather than our current blacklist approach.

Yeah, with this approach we'll scrape baked-in edges out of the release image here, rather than requiring Cincinnati to extract those directly.  The push-to-quay approach is just a cludge until we have a Cincinnati plugin to slurp some yet-to-be-determined JSON output from this repo.